src/fparser/two/tests/test_comments.py

```diff
@@ -242,24 +249,21 @@ def test_prog_comments():
     #   |--> Comment
     #   |--> Main_Program
     #   .    |--> Program_Stmt
-    #   .    |--> Specification_Part
-    #   .    .    \--> Implicit_Part
-    #   .    .         \--> Comment
+    #   .    |--> Comment
     #        |--> Execution_Part
     #        |    |--> Write_Stmt
-    #        |    \--> Comment
+    #        |    |--> Comment
     #   .    .
     #   .
     #   |--> Comment
     from fparser.two.Fortran2003 import Main_Program, Write_Stmt, End_Program_Stmt
```

Comment:
Please move these imports to the top (and check for any subsequent repeats).

Reply:
Moved all imports to the top.